models/event.go

```diff
@@ -1,11 +1,11 @@
 package models
 
-// SearchDataImportModel provides an avro structure for a SearchDataImportModel Called event
-type SearchDataImportModel struct {
+// SearchDataImport provides an avro structure for a SearchDataImport Called event
+type SearchDataImport struct {
 	UID             string               `avro:"uid"`
 	URI             string               `avro:"uri"`
 	Edition         string               `avro:"edition"`
-	DataType        string               `avro:"type"`
+	DataType        string               `avro:"data_type"`
```

Comment:
Why is this changing from `type` to` data_type`?

Reply:
Was this previously broken?

Reply:
I can see that the extractor sets this to `data_type` too - https://github.com/ONSdigital/dp-search-data-extractor/blob/develop/schema/schema.go#L31